File: client/public/Codes/C/CToF.c
#include <stdio.h>

int main()
{
    float c, f;

    printf("Enter Temperature in Celsius :-\n");
    scanf("%f", &c);

    f = c * 9 / 5 + 32;

    printf("Temperature in Farhenheit = %f", f);
    return 0;
}

File: client/public/Codes/Java/CToF.java
import java.util.Scanner;

public class CToF {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the Temperature in Celsius :");
        int celsius = scanner.nextInt();
        double fahrenheit = (1.8 * celsius) + 32;
        System.out.println("Temperature in Fahrenheit : " + fahrenheit);
        scanner.close();
    }
}
